addons/docs/README.md

```diff
@@ -147,65 +140,72 @@ Add the following to your Jest configuration:
 
-`sourceLoaderOptions` is an object for configuring `@storybook/source-loader`. When set to `null` it tells docs not to run the `source-loader` at all, which can be used as an optimization, or if you're already using `source-loader` in your `webpack.config.js`.
+`sourceLoaderOptions` is an object for configuring `@storybook/source-loader`. When set to `null` it tells docs not to run the `source-loader` at all, which can be used as an optimization, or if you're already using `source-loader` in your `main.js`.
 
 ## Manual configuration
 
-If you don't want to use the preset, and prefer to configure "the long way", first register the addon in `.storybook/addons.js`:
+If you don't want to use the preset, and prefer to configure "the long way", first register the addon in `.storybook/main.js`:
 
 ```js
-import '@storybook/addon-docs/register';
+module.exports = {
+  addons: ['@storybook/addon-docs/register']
+};
 ```
 
-Then configure Storybook's webpack loader in `.storybook/webpack.config.js` to understand MDX story files and annotate TS/JS story files with source code using `source-loader`:
+
+Then configure Storybook's webpack loader in `.storybook/main.js` to understand MDX story files and annotate TS/JS story files with source code using `source-loader`:
 
 ```js
 const createCompiler = require('@storybook/addon-docs/mdx-compiler-plugin');
 
-module.exports = async ({ config }) => {
-  config.module.rules.push({
-    test: /\.(stories|story)\.mdx$/,
-    use: [
-      {
-        loader: 'babel-loader',
-        // may or may not need this line depending on your app's setup
-        options: {
-          plugins: ['@babel/plugin-transform-react-jsx'],
+module.exports = {
+  webpack: async config => {
+    config.module.rules.push({
+      test: /\.(stories|story)\.mdx$/,
+      use: [
+        {
+          loader: 'babel-loader',
+          // may or may not need this line depending on your app's setup
+          options: {
+            plugins: ['@babel/plugin-transform-react-jsx'],
+          },
         },
-      },
-      {
-        loader: '@mdx-js/loader',
-        options: {
-          compilers: [createCompiler({})],
+        {
+          loader: '@mdx-js/loader',
+          options: {
+            compilers: [createCompiler({})],
+          },
         },
-      },
-    ],
-  });
-  config.module.rules.push({
-    test: /\.(stories|story)\.[tj]sx?$/,
-    loader: require.resolve('@storybook/source-loader'),
-    exclude: [/node_modules/],
-    enforce: 'pre',
-  });
-  return config;
+      ],
+    });
+    config.module.rules.push({
+      test: /\.(stories|story)\.[tj]sx?$/,
+      loader: require.resolve('@storybook/source-loader'),
+      exclude: [/node_modules/],
+      enforce: 'pre',
+    });
+    return config;
+  }
 };
 ```
 
-Finally, you'll need to set up DocsPage in `.storybook/config.js`:
+Finally, you'll need to set up DocsPage in `.storybook/preview.js`:
```

Comment:
Maybe `preview.js` should have a default export like:

```js
export default {
  parameters: {
     docs: {  ... }
  }
}
```

Reply:
🤔 this is for consistency with CSF & is equiv to global `addParameters` / `addDecorator`(s)?

Reply:
Yes! And it solves the issue with having to import `addDecorators/addParameters` from somewhere.